spanner/api/Spanner.Samples.Tests/SpannerFixture.cs

```diff
@@ -514,6 +514,32 @@ public async Task CreateVenuesTableAndInsertDataAsync(string databaseId)
         }));
     }
 
+    public async Task CreateSingersAndAlbumsTableAsync(string projectId, string instanceId, string databaseId)
+    {
+        using var connection = new SpannerConnection($"Data Source=projects/{projectId}/instances/{instanceId}/databases/{databaseId}");
+;
+        var createSingersTable =
+        @"CREATE TABLE Singers (
+                 SingerId INT64 NOT NULL,
+                 FirstName STRING(1024),
+                 LastName STRING(1024),
+                 ComposerInfo BYTES(MAX),
+                 FullName STRING(2048) AS (ARRAY_TO_STRING([FirstName, LastName], "" "")) STORED
+             ) PRIMARY KEY (SingerId)";
+        // Define create table statement for table #2.
```

Comment:
I'm not sure the comment is really useful here.

Reply:
Removed.